### ISSUE TITLE
Enhancement - Add known flaws file to Python dependencies test and update known flaws file of code vulnerabilities test

### ISSUE THREAD

**Issue by mcarmona99** [wazuh/wazuh-qa#3325]
## Description

We must update the behavior of the [Python dependencies test ](https://github.com/wazuh/wazuh-qa/blob/master/tests/scans/dependencies/test_dependencies.py) and add a file containing known vulnerabilities to be taken into account and that will be skipped if the test detects them.

### Current behavior

Right now, the `tests/scans/dependencies` test checks for vulnerable Python packages in a given `requirements.txt` file.
It could be possible that some of these vulnerable packages detected by the test cannot be upgraded and/or are planned to be upgraded in the future. For instance, https://github.com/wazuh/wazuh/issues/11397#issuecomment-997775563 contains a vulnerability that will not be fixed.

### Expected behavior

These vulnerable packages need to be considered and will be skipped if the test finds them. As said in the Description section, a file containing these known vulnerabilities is needed. The design of the solution will be similar to the one developed in the [code vulnerabilities tests](https://github.com/wazuh/wazuh-qa/blob/master/tests/scans/code_analysis/test_python_flaws.py), where a JSON file with keys `false_positives` and `to_fix` is used. In the case of the dependencies and for instance, we could use a `to_fix` key too, used for dependencies that will be upgraded in the future; and instead of `false_positives`, we could use a `known_flaws`/`known_issues` key to mark vulnerabilities as known but unsolvable.

To sum up, the dependencies test must contain a file to mark vulnerabilities as known.

In addition, we must update both the dependencies and the code vulnerabilities test's known flaws files with the current flaws to be fixed (to_fix tag). Note that these possible vulnerabilities will be investigated in https://github.com/wazuh/wazuh-qa/issues/3319. Both tests must pass by default after the development of this issue.
